synthtool/gcp/templates/node_library/.kokoro/populate-secrets.sh

```diff
@@ -19,25 +19,55 @@ function now { date +"%Y-%m-%d %H:%M:%S" | tr -d '\n' ;}
 function msg { println "$*" >&2 ;}
 function println { printf '%s\n' "$(now) $*" ;}
 
+# Populates requested secrets set in SECRET_MANAGER_KEYS
```

Comment:
Who do we expect to call this script?  When?  Why?  Please add comments around line 15.

Reply:
Comment added

Reply:
Looks good.